internal/esmapping/command_test.go

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2025 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package esmapping
+
+import (
+	"encoding/json"
+	"io"
+	"os"
+	"strings"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/jaegertracing/jaeger/cmd/esmapping-generator/app"
+	"github.com/jaegertracing/jaeger/pkg/testutils"
+)
+
+func TestCommandExecute(t *testing.T) {
+	o := app.Options{
+		Mapping:       "jaeger-span",
+		EsVersion:     7,
+		Shards:        5,
+		Replicas:      1,
+		IndexPrefix:   "jaeger-index",
+		UseILM:        "false",
+		ILMPolicyName: "jaeger-ilm-policy",
+	}
+	cmd := Command(o)
+
+	// Create a pipe to capture stdout
+	r, w, _ := os.Pipe()
+	defer func() {
+		os.Stdout = os.Stderr // Reset stdout
```

Comment:
why stderr? I think it's better to capture original stdout and then restore to the same one.